xee/ext.py

```diff
@@ -637,8 +637,8 @@ def __init__(self, variable_name: str, ee_store: EarthEngineStore):
 
     x_min, y_min, x_max, y_max = self.bounds
 
-    x_size = int(np.ceil((x_max - x_min) / np.abs(self.store.scale_x)))
-    y_size = int(np.ceil((y_max - y_min) / np.abs(self.store.scale_y)))
+    x_size = int(np.round((x_max - x_min) / np.abs(self.store.scale_x)))
+    y_size = int(np.round((y_max - y_min) / np.abs(self.store.scale_y)))
```

Comment:
Did Sai’s modulus trick not work?

I have an internal only test where we want to substitute Xee for computePixels. Does that test pass with this change?

Reply:
I think @shoyer would be good to consult, too.

Reply:
No @alxmrs sai's modulus trick is not working on all the images.
And Yes, that internal test was passed with this change.

Reply:
Then this sounds good to me!